src/components/Books.js

```diff
@@ -113,6 +114,7 @@ function Books() {
         <h1>Add New Book</h1>
         <form onSubmit={handleAddBook}>
```

Comment:
- [ ] Please 🙏🏻 adhere to the project requirements, according to which you should add the `onClick` event on the **Button** component, which will help to add the new book into the `redux` store using the reducer. Right now you are using the `onSubmit` event, which is fine but not fulfilling the requirement. You should create a Button.js file for creating the component and then import that component and add the `click` event to make the things working as it working right now.

### Requirement
![Screenshot from 2023-08-23 21-53-57](https://github.com/MohamedRoshdy2021/Book-store/assets/66476329/c7ad4113-ea5e-4e3f-9794-d95fa9e084dd)


### Caution ⚠️ 
**Please you need to replace your form submit button with the <Button onClick={Your_Event_Callback}> Add Book </Button>**.